activerecord/lib/active_record/connection_adapters/mysql/schema_statements.rb

```diff
@@ -167,6 +167,9 @@ def new_column_from_field(table_name, field)
             elsif type_metadata.extra == "DEFAULT_GENERATED"
               default = +"(#{default})" unless default.start_with?("(")
               default, default_function = nil, default
+            elsif type_metadata.type == :text && default
+              # strip and unescape quotes
+              default = default[1...-1].gsub("\\'", "'")
```

Comment:
Is `gsub` the right operation here? I wonder if this might be costly 🤔

Reply:
Did you mean replace it with `gsub!`? `text` columns with default are not popular enough, I think, and default values in the db most of the time have small values in them.

Reply:
@fatkodima Sorry I was misconnecting the dots here. This code probably isn't being called during a request so maybe we don't have to worry about the performance of `gsub`. Please correct me if I'm wrong! :bow:

Reply:
It is used in https://github.com/rails/rails/blob/8a7a9d0a0b27e113e6d0a6087299c4bd41d29738/activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb#L114-L119
which runs once for each table, when columns list for the first time is needed, and then cached in `SchemaCache` - 
https://github.com/rails/rails/blob/8a7a9d0a0b27e113e6d0a6087299c4bd41d29738/activerecord/lib/active_record/connection_adapters/schema_cache.rb#L110-L114